### DISCUSSION TITLE
wrt32x不能安装使用！是否可以增加适配？谢谢

### DISCUSSION THREAD

**Discussion by plafp** [messense/aliyundrive-webdav#429] [Q&A]
如题

**Comment by plafp** [comment]
核心：arm_cortex-a9_vfpv3-d16

**Reply by messense** [reply]
个人没有使用 wrt32x 的需求不考虑，强需求你可以考虑[付费技术支持](https://afdian.net/item?plan_id=b41c3e6acf6911ec912852540025c377)。
